test/api/amialive.test.js

```diff
@@ -9,16 +9,20 @@ let request;
 let testContainer;
 let softwareUpdateChecker;
 
-jest.mock("../../server_src/services/github-client.service");
 require("../../server_src/services/octofarm-update.service");
 const DITokens = require("../../server_src/container.tokens");
+const awilix = require("awilix");
+const AxiosMock = require("../provisioning/axios.mock");
+const GithubApiServiceMock = require("../provisioning/github-api.mock");
 
 const testRoute = "/api/amialive";
 
 beforeAll(async () => {
   await dbHandler.connect();
   const { server, container } = await setupTestApp();
   testContainer = container;
+  container.register(DITokens.httpClient, awilix.asClass(AxiosMock));
```

Comment:
`httpClient` is the `axios` instance. For now its the `require(...)` directly - this allows us to customize it later with axios instances. For now it's to first see where things are headed in the bigger picture.